src/signTx.c

```diff
@@ -1586,6 +1592,77 @@ static void signTx_handleMintAPDU(uint8_t p2, uint8_t* wireDataBuffer, size_t wi
 	signTxMint_handleAPDU(p2, wireDataBuffer, wireDataSize);
 }
 
+// ========================= SCRIPT DATA HASH ==========================
+
+
+enum {
+	HANDLE_DATUM_HASH_STEP_DISPLAY = 1200,
+	HANDLE_DATUM_HASH_STEP_RESPOND,
+	HANDLE_DATUM_HASH_STEP_INVALID,
```

Comment:
This is not about datum hash.

Reply:
Ah, the smart rename confused the two enums and automatically renamed these as well. Nice.